### DISCUSSION TITLE
想问一下控制次字幕上下移动这个功能有没有办法转移到mpv.net-DW这个版本上使用呢？

### DISCUSSION THREAD

**Discussion by jpACGlover** [dyphire/mpv-config#95] [Q&A] [answered]
我想在https://github.com/diana7127/mpv.net-DW这个MPV里实现移动次字幕的功能，请问有办法吗？
![截图_20240609140902](https://github.com/dyphire/mpv-config/assets/171327971/b7b9217b-84d6-4c9f-a76f-a83839aabd10)

**Comment by zhongfly** [comment, marked as answer]
你去它的input.conf找对应项就行了，复制粘贴
建议还是自己了解下菜单项是怎么编辑的，这是使用菜单最基础的
https://github.com/dyphire/mpv-config/blob/ed38b4f6e3a2181a41255a18ec862c3e287c4e9f/input.conf#L224

**Reply by jpACGlover** [reply]
> 你去它的input.conf找对应项就行了，复制粘贴 建议还是自己了解下菜单项是怎么编辑的，这是使用菜单最基础的
> 
> https://github.com/dyphire/mpv-config/blob/ed38b4f6e3a2181a41255a18ec862c3e287c4e9f/input.conf#L224

成功了，非常感谢！
